tests/flow/test_fulltext_query.py

```diff
@@ -21,13 +21,27 @@ def __init__(self):
     def populate_graph(self):
         graph.query("CALL db.idx.fulltext.createNodeIndex('L1', 'v')")
         graph.query("CALL db.idx.fulltext.createNodeIndex({ label: 'L2', stopwords: ['redis', 'world'] }, 'v')")
+        graph.query("CALL db.idx.fulltext.createNodeIndex('L3', { field: 'v1', weight: 1 }, { field: 'v2', weight: 2 })")
+        graph.query("CALL db.idx.fulltext.createNodeIndex('L4', { field: 'v', phonetic: 'dm:en' })")
+        graph.query("CALL db.idx.fulltext.createNodeIndex('L5', { field: 'v', nostem: true })")
```

Comment:
Do we have tests which verifies the behavior for invalid values for:
 `weight` e.g. 0 & negative values ?
`phonetic` invalid string values / none string values e.g. 3 ?
`nostem` none boolean values?

Invalid invocations of `CALL db.idx.fulltext.createNodeIndex` ?
e.g. `CALL db.idx.fulltext.createNodeIndex('L', { field: 'v', nostem: false, nostem: true })`